src/components-examples/material/slide-toggle/slide-toggle-harness/slide-toggle-harness-example.spec.ts

```diff
@@ -1,13 +1,20 @@
-import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {HarnessLoader} from '@angular/cdk/testing';
 import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {provideZoneChangeDetection} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
 import {MatSlideToggleHarness} from '@angular/material/slide-toggle/testing';
-import {HarnessLoader} from '@angular/cdk/testing';
 import {SlideToggleHarnessExample} from './slide-toggle-harness-example';
 
 describe('SlideToggleHarnessExample', () => {
   let fixture: ComponentFixture<SlideToggleHarnessExample>;
   let loader: HarnessLoader;
 
+  beforeEach(() => {
```

Comment:
Should we combine these consecutive `beforeEach` blocks? I think there are some other cases like this in the other files.